pkg/config/defaults/aemm-metadata-default-values.json

```diff
@@ -157,7 +159,9 @@
         "architecture": "x86_64",
         "instanceType": "m4.xlarge",
         "region": "us-east-1"
-      }
+      },
+      "instance-identity-pkcs": "TESTCSqGSIb3DQEHZqCZPIZCZQExCzZJGgUrDgPCGgUZPIZGCSqGSIb3DQEHZaCZJIZEggHderog\nICJhY2NvdW50SWQiIDogIjUxNTPzNjU5NzP4PCIsCiZgImFyY2hpdGVjdHVyZSIgOiZieDg2XzY0\nIirKICZiYXZhaWxhYmlsaXR5Wm9uZSIgOiZidXPtZWFzdC0xYSIsCiZgImJpbGxpbmdQcm9kdWN0\ncyIgOiGudWxsLZogICJkZXZrYXlQcm9kdWN0Q29kZXPiIDogbnVsbCrKICZibWFya2V0cGxhY2VQ\ncm9kdWN0Q29kZXPiIDogbnVsbCrKICZiaW1hZ2VJZCIgOiZiYW1pLTGhODg3ZTQrPWY3NjU0OTP1\nIirKICZiaW5zdGFuY2VJZCIgOiZiaS0rYjU5YTdiN2NlN2UzYmIrYSIsCiZgImluc3RhbmNlVHlr\nZSIgOiZibTQueGxhcmdlIirKICZia2VybmVsSWQiIDogbnVsbCrKICZicGVuZGluZ1RpbWUiIDog\nIjIrPjZtPDPtPDJUPjZ6PzY6NThaIirKICZicHJpdmF0ZUlrIiZ6ICIxNzIuPzEuPzQuNDPiLZog\nICJyYW1kaXNrSWQiIDogbnVsbCrKICZicmVnaW9uIiZ6ICJ1cy1lYXN0LTEiLZogICJ2ZXJzaW9u\nIiZ6ICIyPDE3LTZ5LTPrIgp9ZZZZZZZZPYIGGTCCZRUCZQEraTGcPQsrCQYDVQQGErJVUzEZPGcG\nZ1UECGPQV2FzaGluZ3RvbiGTdGF0ZTEQPZ4GZ1UEGxPHU2VhdHRsZTEgPG4GZ1UEChPXQW1hem9u\nIFdlYiGTZXJ2aWNlcyGPTEPCCQCWukjZ5V4aZzZJGgUrDgPCGgUZoF0rGZYJKoZIhvcNZQkDPQsG\nCSqGSIb3DQEHZTZcGgkqhkiG9r0GCQUxDxcNPjZrPzZyPjZzNzZ1WjZjGgkqhkiG9r0GCQQxFgQU\nN1xlDhvo6cYuGjXZ+mlTW66Ff8rrCQYHKoZIzjgEZrQrPC4CFQCjzjYV1zGUZUxTf6rGO0en/PxR\n3ZIVZK589qSkEaslLdCzeX2GnQ6dz9UeZZZZZZZZ",
```

Comment:
does this value need to be this huge? Will be nice if it can simplified as its a mocked value.

Reply:
I suppose it doesn't but I was just trying to line it up as closely to a real response as possible, so had the number of characters/content-length match. figured this lives in json and although long/ugly to look at shouldn't be too distracting